nasa/run.sh

```diff
@@ -67,7 +82,15 @@ fi
 # arguments.
 
 # shellcheck disable=SC2086
-"${basedir}"/bin/conda/run.sh python "${basedir}/sar2_d2/cli.py" \
-    --calibration-file "${calibration_file}" \
+"${basedir}"/bin/conda/run.sh python "${basedir}/src/alos1_2_gcov/alos1_to_gcov.py" \
+    --in-file "${in_file}" \
     --bbox ${bbox} \
-    --output-dir "${output_dir}"
+    --out-dir "${output_dir}" \
+    --in_type "${in_type}" \
+    --out_type "${out_type}" \
+    --gcov_posting "${posting}" \
```

Comment:
Just a bit of a nit-pick, but I suggest using dash separators instead of underscores to be consistent with the other option names (i.e., don't mix the 2 conventions -- use one or the other exclusively -- dashes are arguably more common):

```suggestion
    --in-type "${in_type}" \
    --out-type "${out_type}" \
    --gcov-posting "${posting}" \
```